pkgs/applications/virtualization/toolbox/default.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+, go-md2man
+, podman
```

Comment:
```suggestion
```
And remove podman from `buildInputs`. Context: 0c70afa6d1626c9a958cf09a9d27ee39217bc51c